localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```
@@ -18,3 +19,187 @@
+    
+ innovation_footmen:0 "Footmen"
+ innovation_footmen_desc:0 "Advancing with raised shields in close formation, the footmen are proven soldiers of the Eastern Kingdoms. Guarded by their large shields and heavy armor and excellent at hand-to-hand combat, they can easily endure volleys of arrows and break enemy charges."
+ 
+ innovation_forsaken_ranged:0 "Forsaken Ranged Tactics"
+ innovation_forsaken_ranged_desc:0 "With our unique ranged armaments, we have produced regiments like the Dark Rangers."
+    
+ innovation_gladiator:0 "Gladiators"
+ innovation_gladiator_desc:0 "Forced to fight for survival, gladiators are professional warriors, capable of great feats in combat."
+
+ innovation_golem:0 "Golem Construct"
+ innovation_golem_desc:0 "Golems are constructs, magically wrought automatons of bones, flesh, metal, or stone and are generally thought to be mindless. They are built generally as guardians or protectors of a person, place, or thing."
+ 
+ innovation_gryphon:0 "Gryphon Taming"
+ innovation_gryphon_desc:0 "Gryphons are majestic creatures of the skies, and are, much like wyverns, more than just beasts. They both are intelligent as well as cunning. Our tamers have successfully learned how to communicate with and tame these flying creatures."
+ 
+ innovation_night_elf_aerial:0 "Darnassian Aerial Tactics"
+ innovation_night_elf_aerial_desc:0 "With our unique aerial armaments, we have produced regiments like the Faerie Dragons and Chimaeras."
+ 
+ innovation_night_elf_ground:0 "Darnassian Ground Tactics"
+ innovation_night_elf_ground_desc:0 "With our unique ground armaments, we have produced regiments like the Watchers."
+ 
+ innovation_night_elf_mounted:0 "Darnassian Mounted Tactics"
+ innovation_night_elf_mounted_desc:0 "With our unique mounted armaments, we have produced regiments like the Huntress."
+ 
+ innovation_peacekeeper:0 "Peacekeepers"
+ innovation_peacekeeper_desc:0 "The peacekeepers defend us from outside threats with all their might. When needed, these great fighters will wage our wars with unwavering spirit. Peacekeepers are better equipped than most of our soldiers in order for them to overcome the challenges we have to face."
+
+ innovation_proto_drake_rider:0 "Proto Drake Riders"
+ innovation_proto_drake_rider_desc:0 "The proto-dragons descend from elementals who avoided being imprisoned in the Elemental Plane and instead remained on Azeroth, where some of them gradually transformed into creatures of flesh and blood. They are the feral, long-forgotten forebears of the dragon species. Our tamers have successfully learned how to communicate with and tame these flying creatures."
+
+ innovation_sentinel:0 "Sentinels"
+ innovation_sentinel_desc:0 "The Sentinels are composed of devout and well-trained warriors. They are disciplined and are masters of their craft, making them an undeseriable enemy for anyone."
```

Fixed the spelling of undesirable. 
```suggestion
 innovation_sentinel_desc:0 "The Sentinels are composed of devout and well-trained warriors. They are disciplined and are masters of their craft, making them an undesirable enemy for anyone."
```